src/rpc/claimtrie.cpp

```diff
@@ -465,13 +487,16 @@ UniValue getclaimbyid(const UniValue& params, bool fHelp)
     pclaimTrie->getClaimById(claimId, name, claimValue);
     if (claimValue.claimId == claimId)
     {
+        CClaimTrieCache cache(pclaimTrie);
         std::vector<CSupportValue> supports;
-        CAmount effectiveAmount = pclaimTrie->getEffectiveAmountForClaim(name, claimValue.claimId, &supports);
+        CAmount effectiveAmount = cache.getEffectiveAmountForClaim(name, claimValue.claimId, &supports);
 
         std::string sValue;
         CCoinsViewCache coins(pcoinsTip);
         getValueForClaim(coins, claimValue.outPoint, sValue);
         claim.push_back(Pair("name", name));
+        if (cache.shouldNormalize())
```

Comment:
I'm thinking to change this to always return the normalized name, not just after the fork. That we people can adjust to it sooner.